src/BitcoinLib/Responses/GetBlockResponse.cs

```diff
@@ -1,22 +1,18 @@
 ﻿// Copyright (c) 2014 - 2016 George Kimionis
 // See the accompanying file LICENSE for the Software License Aggrement
 
+using BitcoinLib.Responses.SharedComponents;
 using System.Collections.Generic;
 
 namespace BitcoinLib.Responses
 {
-    public class GetBlockResponse
+    public abstract class GetBlockResponseAbstract
```

Comment:
Nitpick: would you care renaming this to `GetBlockResponseBase`?

Reply:
Not a problem